<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginPage">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>设备序列号：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>200</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginIdEdit">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>130</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>请输入序列号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginpwdEdit">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>210</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>请输入密码</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>290</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="registePage">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 127);</string>
      </property>
      <widget class="QLineEdit" name="registepwdEdit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>230</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>输入您的密码</string>
       </property>
      </widget>
      <widget class="QPushButton" name="registeButton">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>300</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>设备序列号：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>220</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="registeIdEdit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>170</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>请输入序列号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>461</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
font: 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>设备不存在，请您注册！</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="dispPage">
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,10">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 255, 255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="portLabel">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Courier New&quot;;</string>
            </property>
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="portLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>发送数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;楷体&quot;;
</string>
            </property>
            <property name="text">
             <string>保存数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>终止</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(184, 255, 222);
font: 12pt &quot;楷体&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashDotLine</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="Wavewidget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
